spec/miq_ae_expression_method_spec.rb

```diff
@@ -0,0 +1,169 @@
+module MiqAeExpressionMethodSpec
+  include MiqAeEngine
+  describe MiqAeExpressionMethod do
+    include Spec::Support::AutomationHelper
+    let(:user) { FactoryGirl.create(:user_with_group) }
+    let!(:vm1) { FactoryGirl.create(:vm, :name => 'test_2.1', :cpu_shares => 400) }
+    let!(:vm2) { FactoryGirl.create(:vm, :name => 'test_3.1', :cpu_shares => 400) }
+    let!(:vm3) { FactoryGirl.create(:vm, :name => 'test_4.2', :cpu_shares => 11) }
+    let(:complex_qs_exp) do
+      {"and" => [{"STARTS WITH" => {"field" => "Vm-name", "value" => :user_input}},
+                 {"ENDS WITH"   => {"field" => "Vm-name", "value" => :user_input}}]}
+    end
+
+    let(:m_params) do
+      {'arg1'       => {'datatype' => 'string', 'default_value' => 'test'},
+       'arg2'       => {'datatype' => 'string', 'default_value' => '1'},
+       'attributes' => {'datatype' => 'array',  'default_value' => 'name'}}
+    end
+
+    let(:vm_search) do
+      {:db         => 'Vm',
+       :expression => complex_qs_exp}.to_yaml
+    end
+
+    before do
+      allow(User).to receive(:server_timezone).and_return("UTC")
+    end
+
+    it "expression_method" do
```

Comment:
Suggest moving these tests into something like a `describe "expression_method" do` block to better group and reduce the duplication in each test title.